### DISCUSSION TITLE
Schedule based scaling ? Holidays / Weekends / Nights ?

### DISCUSSION THREAD

**Discussion by hajdukda** [kedacore/keda#4454] [Q&A / Need Help]
Hi,,

I'm looking for a way to schedule scaling based on public holidays etc.
Can KEDA help me with that - if yes - could you point me to documentation ?

I've noticed CRON scheduler but I see its not recommended for that ?

DH

**Comment by tomkerkhove** [comment]
I guess you are looking for https://github.com/kedacore/keda/issues/3566?
